beams/beams/custom_scripts/sales_invoice/sales_invoice.py

```diff
@@ -1,15 +1 @@
 
-import frappe
-from frappe import _
-
-@frappe.whitelist()
-def validate_sales_order_amount(doc, method):
-    """
-        Method to validate the grand total in  Sales Invoice and grand total in  Quotation.
-    """
```

Comment:
Need to check the Invoice amount calculation from Sales Invoice server side too. Then only manual sales invoice creation can be handled.